conda_env/README.rst

```diff
@@ -67,10 +67,10 @@ Then run this from the command line:
     [      COMPLETE      ] |#################################################| 100%
     #
     # To activate this environment, use:
-    # $ source activate numpy
+    # $ conda activate numpy
```

Comment:
```suggestion
    # $ conda activate stats
```


This change suggestion is from the text above:
"...to create an environment named `stats` with `numpy` and `pandas`..."

Reply:
Good catch!